src/components/Preferences/LocalIdentities.js

```diff
@@ -19,132 +20,250 @@ import {
+        window.URL.createObjectURL(
+          new Blob(
+            [
+              JSON.stringify(
+                identities.filter(({ address }) =>
+                  addressesSelected.get(address)
+                )
+              ),
+            ],
+            { type: 'text/json' }
+          )
+        ),
+      [identities]
+    )
+
+    if (!identities.length) {
+      return <EmptyLocalIdentities onImport={onImport} />
     }
-  }
-  const downloadHref = window.URL.createObjectURL(
-    new Blob([JSON.stringify(identities)], { type: 'text/json' })
-  )
-  // standard: https://en.wikipedia.org/wiki/ISO_8601
-  const today = format(Date.now(), 'yyyy-MM-dd')
-
-  return (
-    <React.Fragment>
-      <Headers>
-        <div>Custom label</div>
-        <div>Address</div>
-      </Headers>
-      <List>
-        {identities.map(({ address, name }) => (
-          <Item key={address}>
-            <Label>{name}</Label>
-            <div>
-              <IdentityBadge
-                entity={address}
-                popoverAction={{
-                  label: 'Edit custom label',
-                  onClick: updateLabel(address),
-                }}
-                popoverTitle={<LocalIdentityPopoverTitle label={name} />}
+
+    return (
+      <React.Fragment>
+        <Headers>
+          <div>
+            {!iOS && (
+              <StyledCheckbox
+                checked={allSelected}
+                onChange={onToggleAll}
+                indeterminate={!allSelected && someSelected}
               />
-            </div>
-          </Item>
-        ))}
-      </List>
-      <Controls>
-        <Import onImport={onImport} />
-        {!iOS && (
-          <StyledExport
-            label="Export labels"
-            mode="secondary"
-            download={`aragon-labels_${dao}_${today}.json`}
-            href={downloadHref}
-          >
-            Export
-          </StyledExport>
-        )}
-        <Button label="Remove labels" mode="outline" onClick={onClearAll}>
-          <IconCross /> Remove all labels
-        </Button>
-      </Controls>
-      <Warning />
-    </React.Fragment>
-  )
-}
+            )}
+            Custom label
+          </div>
+          <div>Address</div>
+        </Headers>
+        <List>
+          {identities.map(({ address, name }) => (
+            <Item key={address}>
+              <Label>
+                {!iOS && (
+                  <StyledCheckbox
+                    checked={addressesSelected.get(address)}
+                    onChange={onToggleAddress(address)}
+                  />
+                )}
+                {name}
+              </Label>
+              <div>
+                <IdentityBadge
+                  entity={address}
+                  popoverAction={{
+                    label: 'Edit custom label',
+                    onClick: updateLabel(address),
+                  }}
+                  popoverTitle={<LocalIdentityPopoverTitle label={name} />}
+                />
```

Comment:
To avoid having to set a mechanism in `LocalIdentityBadge` to not resolve the address:

![image](https://user-images.githubusercontent.com/3072458/57513506-b7173a80-730e-11e9-9a0a-96e9445923df.png)

Reply:
Can we leave a comment to remember it? Otherwise someone might be wondering in the future :D